mitodl/bootcamp-ecommerce
# Moved application submission review fields

#### What are the relevant tickets?
No ticket - supports https://github.com/mitodl/bootcamp-ecommerce/issues/458

#### What's this PR do?
- Moves the review fields and submitted date field from the specific submission models to the `ApplicationStepSubmission` model
- Gets rid of the "waiting" review status for the time being (until a clear need arises)

#### How should this be manually tested?
Visit the admin, make sure you can create/edit `ApplicationStepSubmission` and `VideoInterviewSubmission` objects

#### Any background context you want to provide?
This is a minor change, but when writing the logic for deriving a user's state using their submissions, I realized how much more sense it makes to have the review fields as a part of the base submission model
